src/style.css

```diff
@@ -0,0 +1,274 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 707px;
+  padding: 10px;
+}
+.calendar__day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  background-color: #eee;
+  height: 100px;
+  width: 100px;
+  border: 1px solid black;
+}
+.calendar__day:nth-child(1)::before {
```

Comment:
Instead of manually writing each ::before pseudo-element for each day, use @for loop to generate them.